### DISCUSSION TITLE
"Library View": No context menu in non-leaf directory

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#390] [Fixed Issues]
**rAudio 1 • 202106081
5.10.39-2-ARCH aarch64**

e.g.: There is no longer possible for a "Update database" to be applied of all albums of a specific artist.

**Comment by klassikmann** [comment]
**rAudio 1 • 20210609
5.10.42-1-ARCH aarch64
Fix - Missing context menu in Library**

Not fixed. Same situation as before.

**Comment by klassikmann** [comment]
There seems to be a major regression.
I resampled some FLACs from 24/96 to 24/48 because the spectrum of these files show no energy above about 22kHz. Then I made "Update datase" of the directories with the new content. But after the update the playback window still displays 24/96.

Regarding the second fix od 20210609:
`Fix - Missing icons in Library/Playlist`
This issue still exists either.

**Comment by klassikmann** [comment]
STOP THE PRESS!
Seems to be my fault. More infos soon.

EDIT: 
My statement
> There seems to be a major regression.
> I resampled some FLACs from 24/96 to 24/48 because the spectrum of these files show
> no energy above about 22kHz. Then I made "Update datase" of the directories with the
> new content. But after the update the playback window still displays 24/96.

was wrong. Mea culpa, I forgot to distribute the new resampled FLACs from my development system to production. OMG.

But the other statements persist:
- Fix - Missing context menu in Library
- Fix - Missing icons in Library/Playlist

Both fixes of 20210609 do not work.

**Reply by rern** [reply]
Sorry. Wrong update release. It was the existing, not an update.
202106091 has just published.

**Reply by klassikmann** [reply]
**rAudio 1 • 202106091**

Missing context menu in Library: Fixed
Missing icons in Library/Playlist: Fixed (probably, not tested seriously)

Thank you
